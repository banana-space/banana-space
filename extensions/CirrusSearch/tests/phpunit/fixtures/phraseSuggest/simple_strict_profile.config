{
	"query": "hello wolrd",
	"namespaces": [0],
	"offset": 0,
	"profile": "strict",
	"config" : {
		"CirrusSearchEnablePhraseSuggest": true,
		"CirrusSearchFallbackProfile": "phrase_suggest_strict"
	}
}
