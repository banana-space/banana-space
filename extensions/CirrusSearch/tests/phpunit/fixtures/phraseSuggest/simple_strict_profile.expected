{
    "suggest": {
        "text": "hello wolrd",
        "suggest": {
            "phrase": {
                "field": "suggest",
                "size": 1,
                "max_errors": 2,
                "confidence": 2,
                "real_word_error_likelihood": 0.95,
                "direct_generator": [
                    {
                        "field": "suggest",
                        "suggest_mode": "always",
                        "max_term_freq": 0.5,
                        "min_doc_freq": 0,
                        "prefix_length": 2
                    }
                ],
                "highlight": {
                    "pre_tag": "\ue000",
                    "post_tag": "\ue001"
                },
                "collate": {
                    "query": {
                        "inline": {
                            "multi_match": {
                                "query": "{{suggestion}}",
                                "operator": "or",
                                "minimum_should_match": "3<66%",
                                "type": "cross_fields",
                                "fields": [
                                    "title.plain",
                                    "redirect.title.plain"
                                ]
                            }
                        }
                    }
                },
                "smoothing": {
                    "laplace": {
                        "alpha": 0.3
                    }
                }
            }
        }
    }
}